### ISSUE TITLE
Error Running Tests 

### ISSUE THREAD

**Issue by simongroom** [yonathan06/fastify-typescript-starter#10]
Having run `pnpm install` followed by `pnpm build`, I can get the server running locally and it works as expected.

However, running `pnpm test` or `pnpm test:watch` fails with the following errors:

```bash
Test Files  no tests
   Tests  no tests
   Start at  12:44:06
   Duration  234ms (transform 39ms, setup 0ms, collect 0ms, tests 0ms)

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Errors ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

Vitest caught 2 unhandled errors during the test run.
This might cause false positive tests. Resolve unhandled errors to make sure your tests are not affected.

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Error ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Error: Failed to load url tinyspy (resolved id: tinyspy). Does the file exist?
 ❯ loadAndTransform node_modules/.pnpm/vite@4.5.2_@types+node@18.11.15/node_modules/vite/dist/node/chunks/dep-52909643.js:55015:21
 ❯ ViteNodeServer._transformRequest node_modules/.pnpm/vitest@0.25.8/node_modules/vitest/dist/chunk-vite-node-externalize.b9495318.js:7059:16
 ❯ ViteNodeServer._fetchModule node_modules/.pnpm/vitest@0.25.8/node_modules/vitest/dist/chunk-vite-node-externalize.b9495318.js:7038:17
 ❯ MessagePort.<anonymous> node_modules/.pnpm/vitest@0.25.8/node_modules/vitest/dist/chunk-vite-node-source-map.b4ea5792.js:419:18


⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Error ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Error: Failed to load url tinyspy (resolved id: tinyspy). Does the file exist?
 ❯ loadAndTransform node_modules/.pnpm/vite@4.5.2_@types+node@18.11.15/node_modules/vite/dist/node/chunks/dep-52909643.js:55015:21
 ❯ ViteNodeServer._transformRequest node_modules/.pnpm/vitest@0.25.8/node_modules/vitest/dist/chunk-vite-node-externalize.b9495318.js:7059:16
 ❯ ViteNodeServer._fetchModule node_modules/.pnpm/vitest@0.25.8/node_modules/vitest/dist/chunk-vite-node-externalize.b9495318.js:7038:17
 ❯ MessagePort.<anonymous> node_modules/.pnpm/vitest@0.25.8/node_modules/vitest/dist/chunk-vite-node-source-map.b4ea5792.js:419:18

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 FAIL  Tests failed. Watching for file changes...
       press h to show help, press q to quit
```

Any idea as to what I am missing here to get the tests passing? I am really impressed by this boilerplate - thank you for providing!

**Comment by simongroom** [comment]
It appears that adding vite as a dependency resolves this issue. I also changed to use latest version of each of these and it appears to work:

```bash
 ✓ tests/server.test.ts (1)
 · tests/routes/index.test.ts (1)
 ✓ tests/server.test.ts (1)
 ✓ tests/routes/index.test.ts (1)

 Test Files  2 passed (2)
      Tests  2 passed (2)
   Start at  13:22:26
   Duration  1.39s (transform 59ms, setup 0ms, collect 2.35s, tests 49ms, environment 0ms, prepare 133ms)

```
